scripts/metadata_table.py

```diff
@@ -1266,6 +1279,36 @@ def is_scalar_reference(test_val):
         """Return True iff <test_val> refers to a Fortran scalar."""
         return check_fortran_ref(test_val, None, False) is not None
 
+    def to_html(self, outdir, props):
```

Comment:
I would move this up to where the other active routines are, say right after `write_to_file`.

Reply:
Done.